pebble/src/main/java/com/mitchellbosecke/pebble/template/ScopeImpl.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * This file is part of Pebble.
+ * <p>
+ * Copyright (c) 2014 by Mitchell Bösecke
+ * <p>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+package com.mitchellbosecke.pebble.template;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A scope is a map of variables. A "local" scope ensures that the search for a particular variable
+ * will end at this scope whether or not it was found.
+ *
+ * @author Mitchell
+ */
+public class ScopeImpl implements Scope {
+
+  /**
+   * A "local" scope ensures that the search for a particular variable will end at this scope
+   * whether or not it was found.
+   */
+  private final boolean local;
+
+  /**
+   * The map of variables known at this scope
+   */
+  private final Map<String, Object> backingMap;
```

Comment:
Since this is a map of known variables, why isn't it names something like `knownVariables` or `variablesMap`?